### DISCUSSION TITLE
KIOSK mode entity name edit

### DISCUSSION THREAD

**Discussion by majo551** [tijsverkoyen/HomeAssistant-FusionSolar#136] [Q&A]
Hi,
I just tried the integration in Homeassistant (1.14)  , and I noticed the entity names are rather not user friendly names. Can the names be somehow edited or is it possible the name does not contain the whole technical code in the name? It is hard to find out which of the entity I am looking at: 
![image](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/13347592/22ae3d83-fc96-43bc-aa27-cec5b554b1b9)
Thanks for an advice.

**Comment by tijsverkoyen** [comment]
If you click on it you will see the full name.
In Home Assistant you can also edit the dashboard and alter the name that will be displayed.
